src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -186,6 +186,23 @@ const mockVsCodeApi = () => ({
                       type: "backend"
                     }
                   },
+                  {
+                    name: "Moleculer",
+                    displayName: "Moleculer",
+                    summary: "JavaScript framework",
+                    author: "Various",
+                    order: "1",
+                    icon:
+                      "https://raw.githubusercontent.com/moleculerjs/branding/master/logo/logo-nocircle.svg",
```

Comment:
we should be consistent with the other frameworks and leave this icon empty

Reply:
Nice catch. I've removed